samples/Vento/index.vto

```diff
@@ -0,0 +1,20 @@
+{{ layout "./base.vto" }}
+<div class="doc">
+  {{ include "templates/header.vto" }}
+
+  <main class="doc-content">
+    <header class="doc-header">
+      <h1>
+        {{ title }}
+      </h1>
+      {{ description |> md }}
+
+      {{ include "templates/toc.vto" }}
+    </header>
+
+    <div class="doc-body">
+      {{ content |> md }}
+    </div>
+  </main>
+</div>
+{{ /layout }}
```

Comment:
This is not the sample you reference in the PR template. Either update the template of this sample.

Reply:
Okay, I'll search better examples.